src/content/app/genome-browser/components/browser-track-config/track-config-views/TranscriptTrackConfig.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { RootState } from 'src/store';
+import { useSelector } from 'react-redux';
+
+import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
+
+import { TrackType } from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
+import {
+  getBrowserSelectedCog,
+  getTrackConfigForTrackId
+} from 'src/content/app/genome-browser/state/track-config/trackConfigSelectors';
+
+import useBrowserTrackConfig from '../useBrowserTrackConfig';
+
+import styles from '../BrowserTrackConfig.scss';
+
+export const TranscriptTrackConfig = () => {
+  const selectedCog = useSelector(getBrowserSelectedCog) || '';
```

Comment:
Or maybe we need to add another selector in `trackConfigSelectors` to extract `selectedTrackConfigInfo` rather than calling these two selectors everywhere.